docs/manual/01_install/02_macOSX.md

```diff
@@ -1,49 +1,113 @@
 ---
-sidebar_label: Mac OS X
+sidebar_label: macOS
 ---
-# Installing on Mac OS X
 
-1. Go to the root folder where Ethereal Engine is stored and run
+# Installing on macOS
+
+## Introduction
+
+This guide details the installation process for the iR Engine on macOS devices.
+
+## Prerequisites
+
+Before starting the installation, ensure you have the following tools installed on your system:
```

Comment:
This is more of a stylistic comment, but I tend to prefer switching the order of conditional statements and their resolution.

> Before starting the installation, ensure you have the following tools installed on your system:

> Ensure you have the following tools installed on your system before starting the installation:

I feel like the second flows better when giving direct instructions and stating requirements. But feel free to ignore this comment, as it is more about style than it is about correctness. :writing_hand: